<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HritageQLineEditClass</class>
 <widget class="QMainWindow" name="HritageQLineEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HritageQLineEdit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Afficher_Chaines">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les 10 dernière chaines sauvegardées</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RAZ_Sauvegarde">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remettre à zéro la sauvegarde</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sauvegarder_Chaines">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauvegarder les dernières chaines</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>104</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>QLineEdit_Histo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>QLineEdit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HritageQLineEdit.qrc"/>
 </resources>
 <connections/>
</ui>
